model/database/registry.yaml

```diff
@@ -541,9 +542,50 @@ groups:
               value: "ConsistentPrefix"
               stability: experimental
         stability: experimental
-        brief: Account or request [consistency level](https://learn.microsoft.com/azure/cosmos-db/consistency-levels).
+        brief: >
+          Effective Request [consistency level](https://learn.microsoft.com/azure/cosmos-db/consistency-levels).
+          This represents the first applicable consistency level among those specified by the request or the client.
         examples: ["Eventual", "ConsistentPrefix", "BoundedStaleness", "Strong", "Session"]
 
+  - id: registry.db.cosmosdb.network
+    type: attribute_group
+    display_name: Azure Cosmos DB Network level Attributes
+    stability: experimental
+    brief: >
+      This group defines attributes for Azure Cosmos DB Network call.
+    attributes:
+      - id: db.cosmosdb.network.host
+        type: string
+        stability: experimental
+        brief: Specifies the host address from the Gateway or Backend Service endpoint.
+        examples: ["rntbd://cdb-east-us.azure.document.com"]
+      - id: db.cosmosdb.network.port
+        type: int
+        stability: experimental
+        brief: The port used for the Gateway or Backend Service endpoint.
+        examples: 443
+      - id: db.cosmosdb.network.response.status_code
+        type: int
+        stability: experimental
+        brief: The HTTP status code returned from the network request.
+        examples: 200
+      - id: db.cosmosdb.network.response.sub_status_code
+        type: int
+        stability: experimental
+        brief: The Azure Cosmos DB sub-status code for a network request made during an operation.
+        examples: 1002
+      - id: db.cosmosdb.network.region
+        type: string
```

Comment:
I really don't want to list the region names here because we keep adding new region and it would be very hard to maintain that, along with it, i don't think (at least I am not aware) if we document region list anywhere.

Reply:
how can you stay consistent across languages without have a list documented somewhere?

Reply:
consistency already maintained across SDKs. As soon as we get a new region, we add it in this list https://github.com/Azure/azure-sdk-for-java/blob/ab2fdccc08946c9571424558700fce253118f4da/sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/routing/RegionNameToRegionIdMap.java#L27 similar list is maintained in .Net SDK also.

Why do you want to put region list here, customer/user should not be concerned about the value, as soon as they are getting a value here.

Reply:
Here's the list of values https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/AzureLocation.cs or https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/AzureLocation.cs

You'd also get it if you run `az account list-locations -o table`

<img width="593" alt="image" src="https://github.com/user-attachments/assets/9e309c32-e192-4503-88c3-7bc67e828ef3">

Here's the REST API behind it - `displayName` and `name` are part of this API - https://learn.microsoft.com/en-us/rest/api/subscription/subscriptions/list-locations?view=rest-subscription-2021-10-01&tabs=HTTP#location

As you can see
- there are at least three ways to record region. We need to pick one of them.
- this is not cosmos only concept. there is already an attribute for it - `cloud.region`

https://github.com/open-telemetry/semantic-conventions/blob/43f9e308ada81c5ef6851cd665d95f294b157a0f/model/cloud/registry.yaml#L48

So the proposal is to avoid adding a new attribute and instead 
- reuse `cloud.region`
- we can go ahead and update `cloud.region` description for azure and say that it should match either `displayName` or `name`. I think we should use `name` since it's one-word, lower-case and seems to be more query friendly.

Reply:
I am fine with `cloud.region` as an attribute name but value would be same as `displayName` in above list.

Reply:
I don't think we should put the whole list of Azure regions here. Because we allow new region without SDK upgrade and above list is mostly applicable for Management SDKs. and everytime, we do a region upgrade, we need to make sure this list is upto date for future support)

Reply:
@lmolkova - I agree the frustration that every Azure service defined region names more or less independently and there are different variations. It started because there was no central API/repository/guidance around this i guess, but the fact is that region names are independent for different services and there is no way to change it. You could of course extend the AzureLocation API to contain mappings to the service-specific region names - and especially for configuration that could be useful. But the drawback of using strict enums for region names is that it requires SDK upgrades for every single new Azure region we add. We were in this situation in the V2 .Net SDK (luckily now out of support) and for sure will not go down that direction again - it should be possible to start using a new Azure Region without forcing SDK upgrades. For the telemetry - using a strict enum is a bad idea for the same reason - adding any new value to an enum would require a new version of telemetry. How about making this a string - and referencing a document with the region names/mappings form the doucmentation. The consistency in region names within Cosmos DB across languages comes from the fact that the service requires certain region names to be used - so, the service normalizes there. And the AzureLocation above is by the way also missing regions (UsSec for example or New Zealand north). IMO any reasonable implementation in Azure.Core should get the metadata for regions based off of a service call - a region provider service - we could have all the mappings across sevrices there and provide one central place where new region configuration needs to be added and new regions would become usable without having to just through the hoops of upgrading SDKs.  If there is anyone in Azure Central willing to drive such an API please let me know and I am happy to contribute.

Reply:
As @FabianMeiswinkel  suggested here, we can keep it as of now and once we have the list supported region available publicly..I will update the link here.